### PR TITLE
Documentation and Install script

### PR DESCRIPTION
**Pull request by Akshay-Rohatgi**

### CODE REVIEW

File: misc/install.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+go get "github.com/gin-gonic/gin"
+go get "github.com/google/uuid"
+go get "github.com/pkg/errors"
+go get "go.mongodb.org/mongo-driver/bson"
+go get "go.mongodb.org/mongo-driver/mongo"
+go get "go.mongodb.org/mongo-driver/mongo/options"
+
+echo "[+] Installing mongoDB"
+apt-get install gnupg
+wget -qO - https://www.mongodb.org/static/pgp/server-4.4.asc | sudo apt-key add -
```
Comment (sourque) [inline misc/install.sh:40]: I just `apt install mongodb` haha

File: README.md [hunk 2]
```diff
@@ -2,18 +2,34 @@
...
+touch sarpedon.conf
+./sarpedon # After you finish your sarpedon.conf
 ```
-event = "My Event"
-password = "s3cr3tP4ssw0rd"
 
-[[admin]]
+Example configuration (`sarpedon.conf`):
+
+```toml
```
Comment (sourque) [inline README.md:23]: Nice

**Review by sourque — APPROVED** [review]

File: misc/install.sh (hunk 1 above)
Comment (Akshay-Rohatgi) [reply misc/install.sh:40]: That didnt work for me lol

File: README.md (hunk 2 above)
Comment (Akshay-Rohatgi) [reply README.md:23]: much

**Pull request merged**